src/hubbleds/viewers/hubble_dotplot.py

```diff
@@ -0,0 +1,66 @@
+from cosmicds.viewers import cds_viewer
+from cosmicds.viewers.dotplot.viewer import BqplotDotPlotView
+from cosmicds.viewers.dotplot.state import DotPlotViewerState
+
+from cosmicds.mixins import LineHoverStateMixin, LineHoverViewerMixin
+
+__all__ = [ 'HubbleDotPlotView', 'HubbleDotPlotViewState', 'HubbleDotPlotViewerState']
+
+
+class HubbleDotPlotViewerState(LineHoverStateMixin,DotPlotViewerState):
+    
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        
+
+class HubbleDotPlotViewState(LineHoverViewerMixin,BqplotDotPlotView):
```

Comment:
This shouldn't have `State` in the class name, since it's not a state class, it's a viewer class (that ultimately gets wrapped by `cds_viewer`)